### PR TITLE
Dev project cleaner for cft-dev org

### PR DESCRIPTION
**Pull request by bharathkkb**
This adds auto cleanup functionality to our dev org

### CODE REVIEW

File: infra/terraform/dev-org/dev-project-cleanup/backend.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  backend "gcs" {
+    bucket = "cft-infra-dev-tfstate-test"
```
Comment (bharathkkb) [inline infra/terraform/dev-org/dev-project-cleanup/backend.tf:19]: todo: correct bucket for prod

File: infra/terraform/dev-org/dev-project-cleanup/locals.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  org_id          = "tbd"
```
Comment (bharathkkb) [inline infra/terraform/dev-org/dev-project-cleanup/locals.tf:18]: @morgante can we hard code these?

File: infra/terraform/dev-org/dev-project-cleanup/locals.tf [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  billing_account = "tbd"
+  cleanup_folder  = "tbd"
```
Comment (bharathkkb) [inline infra/terraform/dev-org/dev-project-cleanup/locals.tf:20]: @morgante cleaner function requires all folders to clean to be under to a parent folder. However current dev hierarchy is 
- org
    - user1
    - user2

we can move them to a common folder or we can modify the cleaner function to accept an org.

Comment (morgante) [reply infra/terraform/dev-org/dev-project-cleanup/locals.tf:20]: Let's move them to a common folder.

File: infra/terraform/dev-org/dev-project-cleanup/locals.tf (hunk 2 above)
Comment (morgante) [reply infra/terraform/dev-org/dev-project-cleanup/locals.tf:18]: Yeah that's fine.

File: infra/terraform/dev-org/dev-project-cleanup/locals.tf (hunk 3 above)
Comment (bharathkkb) [reply infra/terraform/dev-org/dev-project-cleanup/locals.tf]: Currently moved mine into the folder. Will announce and move other gradually.

**Comment by bharathkkb** [comment]
@morgante this has been deployed and is working well. I will make an announcement soon to have folks move their folders into the managed folder. I need another stamp to merge this.

**Review by morgante — DISMISSED** [review]

**Comment by bharathkkb** [comment]
This has been applied.

**Pull request merged**